dragonboard/io.py

```diff
@@ -125,6 +144,27 @@ def previous(self):
             raise ValueError('Already at first event')
         return self.next()
 
+    def _update_last_seen(self, event_header):
+        since = np.full(
+            num_channels,
+            np.nan,
+            dtype=[
+                ("low", 'f4', self.roi),
+                ("high", 'f4', self.roi),
+            ]
+        )
+
+        for g, p in stop_cell_map:
+            sc = event_header.stop_cells[g][p]
+
+            since[g][p] = np.roll(self.last_seen[g][p], -sc)[:self.roi]
```

Comment:
We learned in #30 that `np.role` is quite expensive. Is it really necessery here?